utils/tests/crypto.test.js

```diff
@@ -79,16 +81,16 @@ describe('Utils crypto', () => {
       jasmine.DEFAULT_TIMEOUT_INTERVAL = 30 * 1000;
       const mnemonic = 'cram swing cover prefer miss modify ritual silly deliver chunk behind inform able';
 
-      const res = isElectrumVaultMnemonic(mnemonic);
-
+      const res = isElectrumVaultMnemonic(mnemonic, ELECTRUM_VAULT_SEED_PREFIXES.SEED_PREFIX);
       expect(res).toBe(true);
     });
 
     it('should return true for segwit electrum vault mnemonic', () => {
       jasmine.DEFAULT_TIMEOUT_INTERVAL = 30 * 1000;
       const mnemonic = 'grant oval resource roast virtual wine chef inmate attack flip fresh reduce';
 
-      const res = isElectrumVaultMnemonic(mnemonic);
+      console.log('ELECTRUM_VAULT_SEED_PREFIXES.SEED_PREFIX_SEGWIT', ELECTRUM_VAULT_SEED_PREFIXES.SEED_PREFIX_SEGWIT);
```

Comment:
what about using our logger here instead of `console.log()`?

Reply:
at least it's only for debugging purposes, then I'd suggest to remove it

Reply:
unless* (edit function doesn't work :( )

Reply:
yeah, the mistake will be removed in next pr